### ISSUE TITLE
SVN svn:externals properties with specifying a revision gives an error in the UI browser

### ISSUE THREAD

**Issue by Jan-cpu82** [scm-manager/scm-manager#2239]
The SCMMANAGER version is 3.5.0
When i add an svn:externals property to trunk in an repository without specifying a specific revision thru Tortoise svn client it is working fine.
See screenshot for SCMMANAGER, Wiliam is the external folder:
![scmmanager_3](https://github.com/user-attachments/assets/3d3d7560-7ec6-44ce-9925-9ecb9131a8ef)

When i specify a specific revision in the svn:externals property tru for example Tortoise svn client it is working good in the Tortoise browser and cli but not in the SCMMANAGER UI browser which gives an error.

SCMMANAGER screenshot:
![scmmanager_5](https://github.com/user-attachments/assets/d85d0ec6-38e0-437d-93e3-116c4b1440c3)

**Comment by tidiegeler** [comment]
Hi @Jan-cpu82, I've taken a look at the issue in the meanwhile.

What link is set in your svn:externals properties for this folder? It is likely that you took the brank link from the URL bar in your browser. It is not the same format as the link you get in the 'Information' section (the one that is used for a svn checkout).

Generally, it is seen as safe to use the link for the SVN repository from the 'Information' section and attach the patch to the folder thereafter. This did work both for the native SVN CLI and TortoiseSVN on Windows for a branch in an external folder.

However, we could reproduce your problem when picking the URL not from the 'Information' section but directly from the URL bar. We're going to discuss how we want to deal with this problem.

Thanks for your report and I hope you're having a generally good experience with the SCM-Manager.

**Comment by tidiegeler** [comment]
Hello @Jan-cpu82,

thank you for your hint to external revisions in SVN. As far as I could find out, we don't seem to support this feature correctly yet. However, since TortoiseSVN [strongly emphasizes](https://tortoisesvn.net/docs/release/TortoiseSVN_en/tsvn-dug-externals.html) usage of explicit revisions for external references, we are going to consider the implementation of external revisions in one of the upcoming releases. It is expected to take a while until this would be available if we decided on it, though.

**Comment by Jan-cpu82** [comment]
Hello @tidiegeler ,
Sorry for the late response.
I have the following line in my svn:externals property:
https://*url*/scm/repo/Test/Testje2/trunk/Wiliam@3 Wiliam

svn:externals are not only a Tortoise thing but are a standard SVN feature.
See:
https://svnbook.red-bean.com/en/1.7/svn.advanced.externals.html

**Comment by tidiegeler** [comment]
> Hello [@tidiegeler](https://github.com/tidiegeler) , Sorry for the late response. I have the following line in my svn:externals property: https://_url_/scm/repo/Test/Testje2/trunk/Wiliam@3 Wiliam (i replaced my url with url)
> 
> svn:externals are not only a Tortoise thing but are a standard SVN feature. See: https://svnbook.red-bean.com/en/1.7/svn.advanced.externals.html

Yes, I could reproduce the same issue and already found a likely internal cause for it. I would guess the fix will be part of one of the upcoming 3.X releases.

**Comment by Jan-cpu82** [comment]
Ping, i don't think this should be closed yet.
